statsmodels/stats/tests/test_multi.py

```diff
@@ -381,7 +381,7 @@ def test_tukeyhsd():
         [-19.016667, -37.204253, -0.8290806, 0.037710044]])
 
     m_r = [94.39167, 102.54167,  91.13333, 118.20000,  99.18333]
-    myres = tukeyhsd(m_r, 6, 110.8, alpha=0.05, df=4)
+    myres = tukeyhsd(m_r, 6, 110.8254416667, alpha=0.05, df=4)
```

Comment:
This value was originally truncated. With these additional digits, the tolerance can actually be decreased when using SciPy’s Studentized Range. 

```
library(MASS)
options(digits=16)
data("immer", package = "MASS")
immer.aov <- aov((Y1 + Y2)/2 ~ Var + Loc, data = immer)
summary(immer.aov)
```
Prints:
```
            Df          Sum Sq         Mean Sq  F value     Pr(>F)    
Var          4  2654.986166667  663.7465416667  5.98912  0.0024526 ** 
Loc          5 10610.452000000 2122.0904000000 19.14804 5.2119e-07 ***
Residuals   20  2216.508833333  110.8254416667     
```